### PR TITLE
ft: add sold-by card component

### PR DESCRIPTION
**Pull request by SergeInGithub**
# Description

This will add the following component: 
- Sold by card

The mentioned card will show the seller of the product being viewed and it will appear when viewing a specific product

📹   Loom video link: https://www.loom.com/share/fc1cfd243ef64833900b4a4f53033bb0

Fixes # (#21)

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have created a loom video for the new feature or enhancement
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by shyakadavis — COMMENTED** [review]
Great work.

Suggestions:

- Could wrap the links in actual `Link` components that can take dynamic endpoints.
- Make the promotions/discounts standout with greenish or reddish colours.

**Review by musllim — APPROVED** [review]
Looks good

**Review by shyakadavis — DISMISSED** [review]
LGMT.

Be on the look out for any changes to how the routing will be. Especially if this #33 is ever accepted, because, it will introduce abstraction as to how we treat routing.

**Review by emilenfc — DISMISSED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,7 @@ lerna-debug.log*
 build/
 
 node_modules
+coverage
```
Comment (shyakadavis) [inline .gitignore:13]: Not an issue, but II think you added this a second time. It was already there.

File: src/components/shared/SoldByCard.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+function SoldByCard({
+  seller,
+  feedback,
+  sellerItems,
+  contactSeller,
+}: SoldByProps) {
+  return (
+    <div className="bg-white rounded-lg w-52 h-52 mt-2 drop-shadow-lg">
+      <h3 className="text-dark text-lg font-medium ml-4 pt-3">Sold by:</h3>
+      <p className="text-primary font-medium ml-4 mt-2">{seller}</p>
```
Comment (shyakadavis) [inline src/components/shared/SoldByCard.tsx:20]: I think this should also be a link in case a buyer wants to view all products sold by a specific person.

**Review by musllim — APPROVED** [review]
LGTM

File: src/components/shared/SoldByCard.tsx (hunk 2 above)
Comment (shyakadavis) [reply src/components/shared/SoldByCard.tsx:20]: Oh. I was misled then. I saw the primary colour for the text (in the ui), and instantly assumed that it was a clickable link.

**Review by musllim — APPROVED** [review]
LGTM

**Review by shyakadavis — APPROVED** [review]
LGMT

**Review by musllim — APPROVED** [review]
LGTM

**Review by ISHIMWE-Aime — DISMISSED** [review]
You've done great work
LGTM

**Review by emilenfc — APPROVED** [review]
LGTM

**Pull request merged**